modal/requirements/PREVIEW.txt

```diff
@@ -1,16 +1,17 @@
-aiohappyeyeballs==2.4.3
-aiohttp==3.10.8
-aiosignal==1.3.1
-async-timeout==4.0.3 ; python_version < "3.11"
-attrs==24.2.0
-certifi==2024.8.30
-frozenlist==1.4.1
-grpclib==0.4.7
-h2==4.1.0
-hpack==4.0.0
-hyperframe==6.0.1
+aiohappyeyeballs==2.6.1
+aiohttp==3.12.7
+aiosignal==1.3.2
+async-timeout==5.0.1 ; python_version < "3.11"
+attrs==25.3.0
+certifi==2025.4.26
+frozenlist==1.6.0
+grpclib==0.4.8
```

Comment:
Note we currently pin 0.4.7 as a library dependency; we should update that

Reply:
Can we bump the version in this PR?

Reply:
We could or we could do it separately; that change would affect all client upgraders immediately, whereas this PR won't affect anyone who's not opting into the PREVIEW image builder (basically nobody)